pydantic/main.py

```diff
@@ -17,6 +17,9 @@ class BaseConfig:
     allow_extra = False
     fields = None
 
+    # tagged on here to make avoid the need for extra imports, e.g. you can just use `BaseModel.Config.Required`
```

Comment:
does BaseModel.Config.Required exist? Should this comment say `pydantic.Required`